client/app/templates/project.hbs

```diff
@@ -159,6 +159,39 @@
 
     <Messages::Assistance class="large-margin-top" />
 
+    <h3>
+      City Planning Staff
+    </h3>
+    <ul class="no-bullet">
+      {{#each this.project.teamMembers as |teamMember|}}
+        <li class="grid-x medium-margin-bottom">
```

Comment:
Should we put this in its own component?

Reply:
Yes!